routes/api.php

```diff
@@ -4,7 +4,9 @@
     Route::resource('tickets', 'TicketsController', ['except' => 'destroy']);
     Route::post('tickets/{ticket}/comments', 'CommentsController@store');
     Route::post('tickets/{ticket}/assign', 'TicketAssignController@store');
+    Route::post('users/create', 'UsersController@store');
```

Comment:
actually the `create` is not required, a simple post to users will create them

Reply:
Well an endpoint to create and one to list them, I can change the endpoint name in case

Reply:
yes, `POST users` to create one, `GET users` list them

Reply:
`GET users/{user_id}` to the full info of one